README.md

```diff
@@ -41,7 +41,7 @@ Key goals
 
 We aim to provide tools which are:
 
-1. Boosting productiviy 
+1. Boosting productivity while reducing code-bloat
```

Comment:
What's the motivation? I feel like a more concise description is better.

Reply:
I thought a more convincing fact will be handy here.

Reply:
Fair enough, though we tend to focus on pragmatic cost-benefit reasoning. :) "Boosting productivity while reducing maintenance $(cost|burden)" could fit better.